src/integrationTest/java/uk/gov/hmcts/reform/professionalapi/ModifyUserRoleIntegrationTest.java

```diff
@@ -179,6 +180,31 @@ public void ac9_modify_roles_with_prd_admin_role_should_return_500_internal_serv
 
     }
 
+    @Test
+    public void ac10_modify_roles_of_active_users_for_an_organisation_with_invalid_org_id_should_return_400() {
+        String organisationIdentifier = createOrganisationRequest();
+        updateOrganisation(organisationIdentifier, hmctsAdmin, ACTIVE);
+        userProfileCreateUserWireMock(HttpStatus.CREATED);
+
+        userProfileCreateUserWireMock(HttpStatus.CREATED);
+        updateUserProfileRolesMock(HttpStatus.OK);
+
+        NewUserCreationRequest userCreationRequest = aNewUserCreationRequest()
+                .firstName("someName")
+                .lastName("someLastName")
+                .email(randomAlphabetic(5) + "@email.com")
+                .roles(Arrays.asList(puiCaseManager))
+                .jurisdictions(OrganisationFixtures.createJurisdictions())
+                .build();
+
+        Map<String, Object> newUserResponse = professionalReferenceDataClient.addUserToOrganisation(organisationIdentifier, userCreationRequest, hmctsAdmin);
+        String userIdentifier = (String) newUserResponse.get("userIdentifier");
+
+        Map<String, Object> response = professionalReferenceDataClient.modifyUserRolesOfOrganisation(createModifyUserProfileData(), "invalid", userIdentifier, hmctsAdmin);
+        assertThat(response.get("http_status")).isNotNull();
```

Comment:
should pass org  starting with "%" and also try for similar instances like putting $ , * which ever you think will eligible for sql injection.

Reply:
I have addressed this in the following commit: https://github.com/hmcts/rd-professional-api/pull/442/commits/a21711e957ca86cd8d21cf514f627cf4f5326028